packages/web/src/components/gcds-date-modified/gcds-date-modified.css

```diff
@@ -1,6 +1,11 @@
 :host {
   display: block;
-  margin-top: var(--gcds-spacing-500);
+  font-family: var(--gcds-date-modified-font-family);
+  font-size: var(--gcds-date-modified-font-size);
+  font-weight: var(--gcds-date-modified-font-weight);
+  line-height: var(--gcds-date-modified-line-height);
+  color: var(--gcds-date-modified-text);
+  margin-top: var(--gcds-date-modified-margin-top);
```

Comment:
In order to be more multi-language friendly, can you change this either to `margin: var(--gcds-date-modified-margin) 0 0;` or `margin-block-start: var(--gcds-date-modified-margin-top);`